### PR TITLE
Hotfix/Update Deploy/Upgrade/Migrate Scripts

### PR DESCRIPTION
**Pull request by teddy-clapp**
## Description
- Include upgrade and migration scripts for QuadPassport, QuadReader, QuadGovernance
- Update tests
## Checklist
1. [x] Unit Testing?
2. [x] Backward Compatible?
3. [x] Upgrade Safe (storage)?
4. [n/a] Updated Confluence?
5. [n/a] Updated Gitbook?

### CODE REVIEW

File: scripts/tasks/migrationV3.ts [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        const governanceAddr = taskArgs.governance;
+        const startBlock = parseInt(taskArgs.startBlock);
+        const endBlock = parseInt(taskArgs.endBlock);
+        const chunkIndex = parseInt(taskArgs.chunkIndex);
+        const ATTRIBUTE_AML = ethers.utils.id("AML");
+        const ATTRIBUTE_COUNTRY = ethers.utils.id("COUNTRY");
+        const ATTRIBUTE_DID = ethers.utils.id("DID");
+        const ATTRIBUTE_IS_BUSINESS = ethers.utils.id("IS_BUSINESS");
+        const ATTRIBUTE_CRED_PROTOCOL_SCORE = ethers.utils.id("CRED_PROTOCOL_SCORE")
+        const ATTRIBUTE_IS_ACCREDITITED_INVESTOR_US = ethers.utils.id("IS_ACCREDITITED_INVESTOR_US")
```
Comment (faze-quadrata) [inline scripts/tasks/migrationV3.ts:24]: ACCREDITED

File: scripts/tasks/migrationV3.ts [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        const ATTRIBUTE_IS_QUALIFIEDPURCHASER_US = ethers.utils.id("IS_QUALIFIEDPURCHASER_US")
```
Comment (faze-quadrata) [inline scripts/tasks/migrationV3.ts:25]: Doublecheck spelling wtih travis. QUALIFIED_PURCHASER?

File: scripts/tasks/migrationV3.ts [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        const governanceAddr = taskArgs.governance;
+        const preapproved = taskArgs.preapproved.split(",");
+        const mode = taskArgs.mode;
+        const ATTRIBUTE_AML = ethers.utils.id("AML");
+        const ATTRIBUTE_COUNTRY = ethers.utils.id("COUNTRY");
+        const ATTRIBUTE_DID = ethers.utils.id("DID");
+        const ATTRIBUTE_IS_BUSINESS = ethers.utils.id("IS_BUSINESS");
+        const ATTRIBUTE_CRED_PROTOCOL_SCORE = ethers.utils.id("CRED_PROTOCOL_SCORE")
+        const ATTRIBUTE_IS_ACCREDITITED_INVESTOR_US = ethers.utils.id("IS_ACCREDITITED_INVESTOR_US")
+        const ATTRIBUTE_IS_QUALIFIEDPURCHASER_US = ethers.utils.id("IS_QUALIFIEDPURCHASER_US")
```
Comment (faze-quadrata) [inline scripts/tasks/migrationV3.ts:124]: this isnt being used anywhere else right? probably can delete since we wont have this anywhere near ready

Comment (teddy-clapp) [reply scripts/tasks/migrationV3.ts:124]: These are the constants that are used globally throughout the repo, goerli, and Mumbai

File: scripts/tasks/migrationV3.ts (hunk 2 above)
Comment (teddy-clapp) [reply scripts/tasks/migrationV3.ts:25]: I checked it with him

File: scripts/tasks/migrationV3.ts (hunk 1 above)
Comment (teddy-clapp) [reply scripts/tasks/migrationV3.ts:24]: Are you sure? I got this one from Fab

Comment (faze-quadrata) [reply scripts/tasks/migrationV3.ts:24]: typo

Comment (teddy-clapp) [reply scripts/tasks/migrationV3.ts]: fixed

**Review by faze-quadrata — APPROVED** [review]

**Review by faze-quadrata — APPROVED** [review]

**Comment by faze-quadrata** [comment]
reviewed in person. more confident on this appraoch

**Pull request merged**